cachito/common/checksum.py

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import hashlib
+from typing import Optional
+from cachito.errors import UnknownHashAlgorithm
+
+
+def get_hasher(algorithm: Optional[str] = None):
```

Comment:
```suggestion
def get_hasher(algorithm: str = "sha256"):
```
For immutable default values, I prefer to have them right in the signature